internal/pkg/cert/cert.go

```diff
@@ -15,21 +15,110 @@
 package cert
 
 import (
+	"bytes"
 	"crypto/tls"
 	"crypto/x509"
+	"encoding/base64"
 	"os"
 
 	"github.com/lf-edge/ekuiper/internal/conf"
 	"github.com/lf-edge/ekuiper/pkg/cast"
 )
 
+func GenTLSConfig(props map[string]interface{}) (*tls.Config, *TlsConfigurationOptions, error) {
```

Comment:
Looks like the second return value *TlsConfigurationOptions is useless outside. Can we remove it?